setup.sh

```diff
@@ -67,7 +67,13 @@ fi
 if [ ! -z "${NNS_DAPP_RELEASE:-}" ]
 then
   mkdir -p nns-dapp/out
-  curl -L "https://github.com/dfinity/nns-dapp/releases/download/${NNS_DAPP_RELEASE}/nns-dapp_local.wasm" -o nns-dapp/out/nns-dapp.wasm
+  NNS_DAPP_WASM_PREFIX="https://github.com/dfinity/nns-dapp/releases/download/${NNS_DAPP_RELEASE}/"
+  FOUND_LOCAL_NNS_DAPP_WASM=$(curl --silent -I -L "${NNS_DAPP_WASM_PREFIX}/nns-dapp_local.wasm" | grep "^HTTP/2 200")
+  if [ ${FOUND_LOCAL_NNS_DAPP_WASM} ]; then
+    curl -L "${NNS_DAPP_WASM_PREFIX}/nns-dapp_local.wasm" -o nns-dapp/out/nns-dapp.wasm
```

Comment:
Why ever use `nns-dapp_local.wasm` instead of `nns-dapp.wasm`?

Reply:
I thought for some releases there's a `nns-dapp_local.wasm` file and for others it's called `nns-dapp.wasm`. Is this not the case?

Reply:
nns-dapp.wasm is always available and some releases also contain nns-dapp_local.wasm, but now the WASM is supposed to be environment agnostic and thus nns-dapp.wasm should be ok to use